src/editor/ImageViewer.js

```diff
@@ -153,7 +151,7 @@ define(function (require, exports, module) {
         this._naturalHeight = e.currentTarget.naturalHeight;
 
         var extension = FileUtils.getFileExtension(this.file.fullPath);
-        var dimensionString = this._naturalWidth + " &times; " + this._naturalHeight + " " + Strings.UNIT_PIXELS;
+        var dimensionString = this._naturalWidth + " (width) &times; " + this._naturalHeight + " (height) " + Strings.UNIT_PIXELS;
```

Comment:
@humphd Yes, you are right. I did not think that these strings should be localized. Thank you!